server/services/mqtt/lib/publish.js

```diff
@@ -10,7 +10,7 @@ const logger = require('../../../utils/logger');
 function publish(topic, message = undefined) {
```

Comment:
No need to put a default value to "undefined", as if the message is not provided, it'll be undefined.